scripts/tools/footprint_scripts_resistorlike.py

```diff
@@ -280,16 +280,18 @@ def makeResistorAxialHorizontal(seriesname, rm, rmdisp, w, d, ddrill, R_POW, typ
 
     # add model
     if (has3d!=0):
-        kicad_mod.append(Model(filename=lib_name + ".3dshapes/"+footprint_name+".wrl", at=x_3d, scale=s_3d, rotate=[0, 0, 0]))
+        kicad_mod.append(Model(filename="${KISYS3DMOD}/" + lib_name + ".3dshapes/"+footprint_name+".wrl", at=x_3d, scale=s_3d, rotate=[0, 0, 0]))

```

Comment:
Prepend 3D path environment variable to the generated 3D model path as the library name now does not contain such path.